src/jti_background.js

```diff
@@ -213,6 +254,80 @@ function addTemplate (templateName, issueTypeField, projectsField, text, callbac
     });
 }
 
+function addDomain (domainName, callback) {
+    chrome.storage.sync.get(StorageID, function (data) {
+        var domainJSON = {};
+
+        if (data[StorageID]) {
+            domainJSON = data[StorageID];
+        }
+
+        var domainID = getNextID(domainJSON.options.domains);
```

Comment:
Doesn't look like you need the `domainID` variable again so no reason to declare the variable
`var newDomain = {
             'id': getNextID(domainJSON.options.domains),
             'name': domainName
         };`